website/.vite/deps_temp_39fa3b02/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
+}
```

Comment:
Should this file be commited? I don't know much about vite, but it is under `.vite` AND in a directory called temp